packages/ember-testing/lib/helpers.js

```diff
@@ -268,18 +268,19 @@ asyncHelper('visit', visit);
 
 /**
   Clicks an element and triggers any actions triggered by the element's `click`
-  event.
+  event. Accepts `context` as an optional second argument.
 
   Example:
 
   ```javascript
-  click('.some-jQuery-selector').then(function() {
+  click('.some-jQuery-selector', 'body').then(function() {
```

Comment:
Since `body` is the default scope, it seems like we could find a better example to motivate the optional second argument.